### PR TITLE
Add SNR check to foreground when plotting and fix paths

### PR DESCRIPTION
**Pull request by wbenoit26**

### CODE REVIEW

**Comment by wbenoit26** [comment]
Yeah, good idea

File: projects/plots/plots/legacy/main.py [hunk 1]
```diff
@@ -92,7 +92,13 @@ def main(
     foreground = RecoveredInjectionSet.read(foreground)
     # Filter unphysical waveforms generated by IMRPhenomXPHM glitch
     # https://git.ligo.org/reed.essick/rpo4-injection-triage/-/blob/main/rpo4a/README.md?ref_type=heads # noqa
-    foreground = foreground[foreground.snr < 1e4]
+    mask = foreground.snr > 1e4
+    num_unphysical = sum(mask)
+    if num_unphysical > 0:
```
Comment (EthanMarx) [inline projects/plots/plots/legacy/main.py:97]: sorry for nit pick, but I dont think we even need the if statement here right?

Comment (wbenoit26) [reply projects/plots/plots/legacy/main.py:97]: I felt like this was the kind of logging statement that could cause confusion if it reported 0 waveforms removed

Comment (EthanMarx) [reply projects/plots/plots/legacy/main.py:97]: sounds good

**Review by EthanMarx — APPROVED** [review]

**Pull request merged**